src/app/qgsprojectproperties.cpp

```diff
@@ -2641,7 +2641,8 @@ void QgsProjectProperties::onGenerateTsFileButton() const
                             "- translate strings\n"
                             "- save it with the postfix of the target language (eg. de)\n"
                             "- release to get qm file including postfix (eg. aproject_de.qm)\n"
-                            "When you open it again in QGIS having set the target language (de), the project will be translated and saved with postfix (eg. aproject_de.qgs)." ).arg( l ) ) ;
+                            "Open the original QGIS file (eg aproject.qgs), if your QGIS is set to use a specific language and the QM file for that language is found, the translated QGIS project will be generated on the fly.\n"
```

Comment:
Iirc it's generated on next save?

Reply:
No, it's generated, redirected and saved when you open it.

I just opened `my_project.qgs` but my QGIS is in French, the title is `my_project_fr.qgs` (in the window title, project properties as well and in the file manager).

Reply:
👍